api/envoy/extensions/local_address_selectors/linux_network_namespace/v3/config.proto

```diff
@@ -0,0 +1,21 @@
+syntax = "proto3";
+
+package envoy.extensions.local_address_selectors.linux_network_namespace.v3;
+
+import "udpa/annotations/status.proto";
+
+option java_package = "io.envoyproxy.envoy.extensions.local_address_selectors.linux_network_namespace.v3";
+option java_outer_classname = "ConfigProto";
+option java_multiple_files = true;
+option go_package = "github.com/envoyproxy/go-control-plane/envoy/extensions/local_address_selectors/linux_network_namespace/v3;linux_network_namespacev3";
+option (udpa.annotations.file_status).package_version_status = ACTIVE;
+
+// [#protodoc-title: Linux Network Namespace Local Address Selector]
+// [#extension: envoy.upstream.local_address_selector.linux_network_namespace]
+
+// Overrides the upstream Linux network namespace using a filter state object
+// with the key ``envoy.network.upstream_network_namespace`` passed from the
+// downstream. The override applies over the :ref:`default address selector
+// <extension_envoy.upstream.local_address_selector.default_local_address_selector>`
```

Comment:
Note for reviewers: I was about to put an inner Any for a nested selector like we do with transport sockets, but it turns the factory drops the proto, so that requires more refactoring that needs a follow-up.